### PR TITLE
Prepare fork for use

### PR DESCRIPTION
**Pull request by dsp25no**

### CODE REVIEW

**Comment by dsp25no** [comment]
С фаззингом были проблемы, почему фаззер падал на каких-то непонятных тестовых примерах. Пришлось фаззить через костыль:
```shell
while true
do
  go test -fuzz=. 2>&1 | tee /tmp/fuzz.log | grep "fuzzing process hung or terminated unexpectedly" -q && bash -c 'echo "Fuzzing failed. Restart" && rm -rf testdata/fuzz/FuzzUnserializer' || break ;
done
```

File: php_serialize/unserialize.go [hunk 1]
```diff
@@ -132,14 +132,18 @@ func (self *UnSerializer) decodeString(left, right rune, isFinal bool) PhpValue
 	self.expect(left)
 
 	if strLen > 0 {
-		buf := make([]byte, strLen, strLen)
-		if readLen, err = self.r.Read(buf); err != nil {
-			self.saveError(fmt.Errorf("php_serialize: Error while reading string value: %v", err))
+		if strLen > UNSERIALIZABLE_OBJECT_MAX_LEN {
```
Comment (WGH-) [inline php_serialize/unserialize.go]: Do large objects cause  slowdown, unbounded memory consumption, or panics?

File: fuzz.go [hunk 2]
```diff
@@ -1,6 +1,7 @@
+//go:build gofuzz
 // +build gofuzz
```
Comment (WGH-) [inline fuzz.go]: BTW, now that we've switched to built-in `testing` fuzzer, maybe it's time to delete gofuzz support, and move its testcases to `FuzzUnserializer` seed input?

File: php_serialize/unserialize.go [hunk 3]
```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 )
 
-const UNSERIALIZABLE_OBJECT_MAX_LEN = 10 * 1024 * 1024
+const UNSERIALIZABLE_OBJECT_MAX_SIZE_DEFAULT = 10 * 1024 * 1024
```
Comment (WGH-) [inline php_serialize/unserialize.go:11]: Snake case should not be used in Go

**Review by WGH- — CHANGES_REQUESTED** [review]

File: php_serialize/unserialize.go (hunk 1 above)
Comment (dsp25no) [reply php_serialize/unserialize.go]: Panics

**Pull request merged**